src/Admin/OptionsMenu.php

```diff
@@ -186,6 +186,24 @@ public function screen_handle() {
 		return sprintf( 'toplevel_page_%s', $this->get_menu_slug() );
 	}
 
+	/**
+	 * Gets analytics vendors list from data directory.
+	 */
+	public function get_analytics_vendors() {
+		$vendors_list_file = AMP__DIR__ . '/includes/ecosystem-data/analytics-vendors.php';
+
+		if ( ! file_exists( $vendors_list_file ) ) {
+			return []; // @codeCoverageIgnore
+		}
+
+		// Fetches analytics vendors list as a array.
+		$vendors_list = require $vendors_list_file;
+
+		$error = json_last_error();
+
+		return ! empty( $error ) ? [] : $vendors_list;
```

Comment:
Just realized that the JSON code is obsolete since a PHP file is now being written.

```suggestion
		return require $vendors_list_file;
```

Reply:
Maybe a bit easier on the eyes:

```suggestion
		$vendors_list = require $vendors_list_file;

		return $vendors_list;
```